src/cpu_shader/flatten.rs

```diff
@@ -151,12 +311,176 @@ fn flatten_cubic(cubic: Cubic, line_ix: &mut usize, lines: &mut [LineSoup]) {
     }
 }
 
+fn flatten_arc(
+    path_ix: u32,
+    begin: Vec2,
+    end: Vec2,
+    center: Vec2,
+    angle: f32,
+    transform: &Transform,
+    line_ix: &mut usize,
+    lines: &mut [LineSoup],
+    bbox: &mut IntBbox,
+) {
+    let mut p0 = transform.apply(begin);
+    let mut r = begin - center;
+    let tol: f32 = 0.5;
```

Comment:
This seems a little loose to me, I've generally been using 0.25. I'm also concerned that it doesn't match the flattening tolerance for curves in general.

This might be a good time for a heads up: I am not confident that the tolerance value in the existing code is accurately calibrated against the actual Fréchet distance between flattened and source curve; I may have let a constant factor of error enter. But the current work I'm doing *does* calibrate that, so when I land that I will be systematic.

Reply:
With the new derivation, 0.25 leads to visible line segments. 0.1 looks decent enough to me. I'm worried that anything lower will blow up the line count but you should probably take a look and make a judgement as to whether this tolerance is visually satisfactory.